packages/typescript/package.json

```diff
@@ -6,7 +6,7 @@
   "homepage": "https://github.com/getsentry/raven-js/tree/master/packages/typescript",
   "author": "Sentry",
   "license": "BSD-3-Clause",
-  "private _": false,
+  "private": false,
```

Comment:
Since this was broken for quite some time, is this key actually relevant? Isn't `"private": false` the default?

Reply:
Correct